### ISSUE TITLE
Cannot install periodictable 1.7.0

### ISSUE THREAD

**Issue by Andrew-S-Rosen** [python-periodictable/periodictable#80]
I am trying to `pip` install `periodictable==1.7.0` in a clean Python 3.11 Conda environment on a Linux HPC machine but am getting the following error. I do not get this error with 1.6.1 

```
pip install periodictable
Collecting periodictable
  Using cached periodictable-1.7.0.tar.gz (1.0 MB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [1 lines of output]
      ERROR: Can not execute `setup.py` since setuptools is not available in the build environment.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed
× Encountered error while generating package metadata.
╰─> See above for output.
note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by Andrew-S-Rosen** [comment]
Looks like this is somewhat of a "me" problem but also a `periodictable` issue that can be resolved. See https://github.com/pkienzle/periodictable/issues/80. By default, the Python environment does not have `setuptools` yet this is imported in `setup.py`. If a `pyproject.toml` were used, this would not be a problem.

**Comment by lucas-wilkins** [comment]
It's not a ""me" problem" - it doesn't work at all with python 3.12.

**Comment by pkienzle** [comment]
Closing because we've released 2.0.
